README.md

```diff
@@ -1,16 +1,34 @@
-<div align="center">
-  <a href="https://nostalgic-css.github.io/NES.css/" target="_blank"><img src="https://user-images.githubusercontent.com/5305599/49061716-da649680-f254-11e8-9a89-d95a7407ec6a.png" alt="NES.css: NES-style  CSS framework" style="max-width:100%;" width="600" height="315"></a>
+<div style="text-align: center;">
```

Comment:
Changing `<div align="center">` to `<div style="text-align: center;">` causes it to bug:

![image](https://user-images.githubusercontent.com/22016005/49832257-467aea80-fd7d-11e8-9205-37c50b3b2f07.png)